pkg/ecdsa/tss/keygen.go

```diff
@@ -18,10 +18,13 @@ const (
 // It times out after 90 seconds if the required parameters could not be generated.
 // It is possible to generate the parameters way ahead of the TSS protocol
 // execution.
-// TODO: Consider pre-generating parameters to a pool and use them on protocol
-// start.
 func GenerateTSSPreParams() (*keygen.LocalPreParams, error) {
-	preParams, err := keygen.GeneratePreParams(preParamsGenerationTimeout)
+	// As a workaround for a bug in tss-lib we have to provide `optionalConcurrency`
+	// parameter with value of at least `3` so the underlying goroutines are
+	// executed on machines with less than 3 CPU.
+	//
+	// See: https://github.com/binance-chain/tss-lib/issues/93
+	preParams, err := keygen.GeneratePreParams(preParamsGenerationTimeout, 3)
```

Comment:
Why don't we make it configurable? With hardcoded `3`, even if I want to dedicate more nodes, I cannot. 
I understand there is a bug in `tss-lib` so maybe we could add a reference to issue 93 in the sample config, explain this parameter does not reflect the number of CPUs but is used as `concurrency / 3 * 2` and let operator to decide?